manifests/m/Microsoft/VisualStudioCode/1.69.0/Microsoft.VisualStudioCode.installer.yaml

```diff
@@ -0,0 +1,34 @@
+PackageIdentifier: Microsoft.VisualStudioCode
+PackageVersion: 1.69.0
+Commands:
+  - code
+InstallerType: inno
+InstallerSwitches:
+  Custom: /mergetasks=!runcode
+Installers:
+  - Architecture: arm64
+    Scope: machine
+    InstallerUrl: 'https://az764295.vo.msecnd.net/stable/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/VSCodeSetup-arm64-1.69.0.exe'
+    InstallerSha256: 9ab6f6c919e9e158ae5a7abcaf7d3f3ff1c9c6af76888d742749b2baeccbe3b5
+  - Architecture: arm64
+    Scope: user
+    InstallerUrl: 'https://az764295.vo.msecnd.net/stable/30d9c6cd9483b2cc586687151bcbcd635f373630/VSCodeUserSetup-arm64-1.68.1.exe'
```

Comment:
@lszomoru I missed this when I was reviewing, but can you make another PR that makes it so all of the links are for 1.69.0? Some of these are still for 1.68.1.

Reply:
https://github.com/microsoft/winget-pkgs/pull/65441